[FEATURE] E-constellatio description (#1)
E-constellatio/.github: profile/README.md

<div align="center">
    <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/E-constellatio.svg" alt="Logo" height="150px">
    <h1>

>         Welcome in E-constellatio Github organization

Reviewer: ```suggestion
        Welcome to the E-constellatio's Github organization
```

    <img src="https://media.giphy.com/media/hvRJCLFzcasrR4ia7z/giphy.gif" alt="Hi hand" width="30px"/>
    </h1>
</div>

> This organization contain repositories of E-constellatio website. Repositories of other products can be found

Reviewer: Since the website for E-constellatio is planned to not hold advanced logics, shouldn't we only have one repository ? 
```suggestion
This organization contains repositories of E-constellatio website. Repositories of other products can be found
```

> This organization contains repositories of E-constellatio website. Repositories which hold products of Correlatio can be

Reviewer: it's either `the E-constellatio website` or `E-constellatio's websites` :)

> in [Libra foundation organization](https://github.com/Libra-foundation)

Reviewer: I do not much like how the phrasing groups Libra and correlatio, those two entities serve very different purposes.

> found in [Correlatio organization](https://github.com/Correlatio-company). All open source projects can be found

Reviewer: ```suggestion
found in the [Correlatio organization](https://github.com/Correlatio-company). All our open sources projects can be found
```

> in [Libra foundation organization](https://github.com/Libra-foundation).

Reviewer: ```suggestion
in the [Libra foundation organization](https://github.com/Libra-foundation).
```

<h2 align="center">
    What is E-constellatio ?
</h2>

<p>

>     E-constellatio is a French and young company based in Belfort created by 3 friends. Company is mainly specialized in digital solutions, and is divided in 3 branches with their field of activity.

Reviewer: The start of the second sentence looks odd, did you forget a word or two ?

>     E-constellatio is a French and young company based in Belfort created by 3 friends specialized in digital solutions. E-constellatio iss divided in 3 branches with their field of activity.

Reviewer: ```suggestion
    E-constellatio is a French and young company based in Belfort created by 3 friends specialized in digital solutions. E-constellatio is divided in 3 branches with their field of activity.
```

too many `s` this time.

</p>

<h2 align="center">
    Branches
</h2>

<div align="center">
    <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/LuxDigitalis.svg" alt="Lux digitalis logo" height="50"/>
    <details>
        <summary>
            <h3>
                Lux Digitalis
            </h3>
        </summary>
        <div align="left">
            <p>

>                 Lux Digitalis is the E-constellatio branch which offer digital consulting and freelancing in web design, integration (front end and back end), AI and cloud infrastructure. Its strong experience in web development is showing through products offered by other E-constellatio branches. We provide skilled people who mastered in a plenty of technologies such as : 

Reviewer: ```suggestion
                Lux Digitalis is the E-constellatio's branch which offer digital consulting and freelancing in web design, integration (front end and back end), AI and cloud infrastructure. Its strong experience in web development is showing through products offered by other E-constellatio branches. We provide skilled people who mastered plenty of technologies such as : 
```

            </p>
        </div>

>         <a href="https://www.typescriptlang.org/">

Reviewer: We might want to do a map of our competencies to better dispatch the work, and ensure we didn't forget any of our strong points.

Reviewer 2: Yes, I totally agree, I will plane a summit with all employees.

        </a>
        <a href="https://www.w3.org/html/">
            <img src="https://github.com/devicons/devicon/blob/master/icons/html5/html5-original.svg" alt="HTML" height="50">
        </a>
        <a href="https://www.w3.org/Style/CSS/Overview.en.html">
            <img src="https://github.com/devicons/devicon/blob/master/icons/css3/css3-original.svg" alt="CSS" height="50">
        </a>
        <a href="https://www.mysql.com/fr/">
            <img src="https://github.com/devicons/devicon/blob/master/icons/mysql/mysql-original.svg" alt="MySQL" height="50">
        </a>
        <a href="https://mui.com/">
            <img src="https://github.com/devicons/devicon/blob/master/icons/materialui/materialui-plain.svg" alt="Mui" height="50">
        </a>
        <a href="https://azure.microsoft.com/fr-fr">
            <img src="https://github.com/devicons/devicon/blob/master/icons/azure/azure-original.svg" alt="Azure" height="50">
        </a>
        <a href="https://www.terraform.io/">
            <img src="https://github.com/devicons/devicon/blob/master/icons/terraform/terraform-original.svg" alt="Terraform" height="50">
        </a>
        <a href="https://eslint.org/">

>             <img src="https://github.com/devicons/devicon/blob/master/icons/eslint/eslint-original.svg" alt="ESLint" height="50">

Reviewer: Lol ! xD

        </a>
    </details>
</div>

<div align="center">
    <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/Puff.svg" alt="Correlatio logo" height="50"/>
    <details>
        <summary>
            <h3>
                Correlatio
            </h3>
        </summary>
        <div align="left">
            <p>

>                 Correlatio offer large amount of statistical computation using AI based on open data of French government and other open statistics foundation.

Reviewer: Not sure if it is smart to write in big where we grab our data from …
I am also not sure how to call what we plan to produce for Crimen, I put indexes because this won't be a simple statical product ... 
```suggestion
                Correlatio offer large amount of statistical computation and AI based indexes based on open data of French government and other open statistics foundation.
```

            </p>
        </div>
        <a href="">

>             <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/Crimen.svg" alt="Crimen logo" height="50">

Reviewer: We should hold a repo for all of our logos, this way if we need to update one or replace a temporary placeholder we only would need one change.

Reviewer 2: Yes, I think it's a good thing. In which `organization` I create it ?

Reviewer: Well, I think the best option would be in the e-constellatio one. I do not see anywhere else to put it.

Reviewer 2: Okay, I will create it and refactor file which use logos.

        </a>
    </details>
</div>

<div align="center">
    <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/Puff.svg" alt="Libra logo" height="50"/>
    <details>
        <summary>
            <h3>
                Libra
            </h3>
        </summary>
        <div align="left">
            <p>

>                 Libra offers abundant free and open digital solutions like softwares, websites, servers, template etc. Open products :

Reviewer: ```suggestion
                Libra offers abundant free and open digital solutions like softwares, websites, servers, templates etc. Open products :
```

            </p>
        </div>
        <a href="https://github.com/Libra-foundation/Collaboratio">
            <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/Collaboratio.svg" alt="Collaboratio logo" height="50">
        </a>
        <a href="https://github.com/Libra-foundation/react-template">
            <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/ReactTemplate.svg" alt="React template logo" height="50">
        </a>

>         <a href="https://github.com/Libra-foundation/python-template">

Reviewer: The UCS is missing from the list.

Reviewer 2: Yes, that's normal because the project is currently empty. I will add it during a refactor.

            <img src="https://github.com/E-constellatio/.github/blob/main/profile/resources/PythonTemplate.svg" alt="Python template logo" height="50">
        </a>
    </details>
</div>

<h2 align="center">
    Employees
</h2>

<div align="center">
    <a href="https://github.com/galyfray">

>         <img src="https://avatars.githubusercontent.com/u/37240229?v=4" alt="Galyfray profile image" height="50">

Reviewer: ```suggestion
        <img src="https://avatars.githubusercontent.com/u/37240229?v=4" alt="Galyfray's profile image" height="50">
```

    </a>
    <a href="https://github.com/Beladric">

>         <img src="https://avatars.githubusercontent.com/u/37240228?v=4" alt="Beladric profile image" height="50">

Reviewer: ```suggestion
        <img src="https://avatars.githubusercontent.com/u/37240228?v=4" alt="Beladric's profile image" height="50">
```

    </a>
    <a href="https://github.com/mnoel2000">

>         <img src="https://avatars.githubusercontent.com/u/95273889?v=4" alt="mnoel2000 profile image" height="50"> 

Reviewer: ```suggestion
        <img src="https://avatars.githubusercontent.com/u/95273889?v=4" alt="mnoel2000's profile image" height="50"> 
```
